### DISCUSSION TITLE
Use kOmegaSSTFieldInversion to test the case"PeriodicHill_FieldInversion".There have been some issues.

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#599] [General]
During the calculation process, A prompt of "Invalid values found. Return primal failure and reset the states to their initial values" appears, how can I solve this problem?
![Snipaste_2024-03-04_15-50-43](https://github.com/mdolab/dafoam/assets/102606153/1443a25d-64ff-4ad9-88a1-6e7290f66d25)

**Comment by friedenhe** [comment]
This is a new feature that will automatically check if flow states are valid. From the screenshot you shared, the omega residual is very large, so please visualize the omega field in paraview. I suspect some of them are very large. Check here for how to fix the flow convergence with the SST model: https://github.com/mdolab/dafoam/issues/483

**Reply by 15175829310** [reply]
Thank you for your reply. It was very helpful. I'll check.

**Reply by uttamcadambi07** [reply]
Ah nice that I syumbled upon this thread. I am currently working on this too but for an airfoil case and I can confirm that the changes you suggested @friedenhe  works well. Thanks a lot!
